pkgs/top-level/default.nix

```diff
@@ -57,8 +57,8 @@ let
   platform = if platform_ != null then platform_
     else config.platform or platformAuto;
 
-  topLevelArguments = {
-    inherit system bootStdenv noSysDirs config crossSystem platform lib;
```

Comment:
Sorry for not commenting on that in the commit message.  I changed `topLevelArguments` to `topLevelArgs` to be more consistent with other variables in the same file that are named `*Args`.  

I took `lib` out of that set because `lib` is not actually a top-level argument; it's not argument to `top-level/default.nix`.  Also, I needed to omit lib from that set on this line:

    allPackages = args: import ./. (topLevelArgs // args);

I hope that makes sense.